samples/charts/sparkline/display-area/Pages/SparklineDisplayArea.razor

```diff
@@ -0,0 +1,98 @@
+﻿@page "/SparklineDisplayArea"
+
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+
+<div class="igContainer"> 
+
+    <div class="igComponent">
+
+        @if (DataSource != null)
+        {
+            <section>Revenue Sparkline</section>
+            <Sparkline Height="100%"
+                       Width="100%"
+                       DataSource="DataSource"
+                       DisplayType="SparklineDisplayType.Area"
+                       ValueMemberPath="Value"
+                       LabelMemberPath="Angle"
+                       Minimum="-1" Maximum="1">
+            </Sparkline>
+
+            <section>Income Sparkline</section>
+            <Sparkline Height="100%"
+                       Width="100%"
+                       DataSource="DataSource"
+                       DisplayType="SparklineDisplayType.Area"
+                       ValueMemberPath="Income"
+                       LabelMemberPath="Angle"
+                       Minimum="-1" Maximum="1">
+            </Sparkline>
+
+            <section>Expanse Sparkline</section>
+            <Sparkline Height="100%"
+                       Width="100%"
+                       DataSource="DataSource"
+                       DisplayType="SparklineDisplayType.Area"
+                       ValueMemberPath="Expanse"
+                       LabelMemberPath="Angle"
+                       Minimum="-1" Maximum="1">
+            </Sparkline>
+           
+        }
+
+    </div>
+</div>
+
+@code {
+
+    private List<DataItem> DataSource;
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        SparklineModule.Register(InfragisticsBlazor);
+
+        var dataItems = new List<DataItem>();
+
+        var index = 0;
+        var min = 1000.0;
+        var max = -1000.0;
+
+        for (var angle = 0; angle <= 360 * 4.0; angle += 5)
+        {
+            var v1 = Math.Sin(angle * Math.PI / 180);
+            var v2 = Math.Sin(3 * angle * Math.PI / 180) / 3;
+            var revenu = v1 + v2;
+
+            var expanse = revenu < 0 ? revenu : 0;
+            var income = revenu > 0 ? revenu : 0;
+
+           dataItems.Add(new DataItem()
+            {
+                Index = index++,
+                Angle = angle,
+                // Value = v1 + v2
+                Value = revenu,
+                Expanse = expanse ,
+                Income=income
+            });
+            min = Math.Min(min, v1 + v2);
+            max = Math.Max(max, v1 + v2);
+        }
```

Comment:
For all Sparkline* Type samples, move this for loop to a new file SparklineData.cs and place it in new class SparklineData and add DataItem class to SparklineData.cs to file.